modules/react/text/lib/LabelText.tsx

```diff
@@ -1,21 +1,19 @@
 import * as React from 'react';
 import {Property} from 'csstype';
 import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
-import {Text, TextProps} from '@workday/canvas-kit-react/text';
+import {Text, TextProps} from './Text';
 import {inputColors} from '@workday/canvas-kit-react/tokens';
 
 export interface TypeLabelProps extends TextProps {
   cursor?: Property.Cursor;
   disabled?: boolean;
 }
 
-const StyledLabel = styled(Text.as('label'))<StyledType & TypeLabelProps>(
```

Comment:
Why are we removing the `.as('label')` here?

Reply:
i think this was just clean up, since `createComponent` and `StyledType` handles this for us.